recipes/magma/thread_queue.patch

```diff
@@ -0,0 +1,20 @@
+--- control/thread_queue.cpp	2016-08-30 06:37:49.000000000 -0700
++++ control/thread_queue.cpp	2016-10-10 19:47:28.911580965 -0700
+@@ -15,7 +15,7 @@
+ {
+     if ( err != 0 ) {
+         fprintf( stderr, "Error: %s (%d)\n", strerror(err), err );
+-        throw std::exception();
++        // throw std::exception();
```

Comment:
What’s the reason for this patch? It looks like it just changes the functionality of the package

Reply:
I'm not sure. I took this recipe from AnacondaRecipes and pytorch recipes and both have it. I'd rather keep it as they know what they are doing and this package is needed for pytorch.

Reply:
That’s good enough for me